Asgardia.Constitution.Proposal.txt

```diff
@@ -342,18 +343,18 @@ Article 31. Legal System and Branch structures
+§ 4. The Head of Nation role shall be abolished upon the adoption of this Constitution. The current Head of Nation, as defined by Decree # 1, shall be requested to take the honourable role of Head of State. After five (5) Earth-years, the first Head of State shall agree to step down unless voted to a second term in office. In the event the Head of Nation declines to become the Head of State, up to five (5) Candidates shall be vetted by special committee appointed by the Head of Nation, within thirty (30) Earth-days, and hold a general Election where the candidate with the most votes shall become the Head of State. In the case where two or more Candidates have the same number of Votes, the Head of Nation shall cast a tie-breaker vote.
+§ 5. The second and subsequent Heads of State may be any Asgardian Citizen that submits his/her candidacy and qualifications to the Parliament for consideration. The Parliament shall appoint a committee of no more than seven (7) members to confirm the qualifications and, upon committee approval, provide the recommendation of the Candidates to the Council of Supreme Values. The Council will further interview the Candidates and confirm their fitness to lead the honourable office as defined by the Supreme Values of Asgardia. Three Earth-months prior to the conclusion of the current Head of State term, the Council of Supreme Values shall present the Candidates to the public for a final vote. There will be a maximum of five (5) candidates including the current Head of State, if he/she has not already served a second term. Two Earth-months prior to the conclusion of the current Head of State, the general election shall commence. Citizen votes shall be collected over a period of 48 Earth-hours and the candidate with the most votes shall be the next Head of State. In the even that two or more Candidates have the same number of votes, the Chairperson of the Council of the Supreme Values shall cast the tie-breaker vote.

```

Comment:
I was thinking about the tie-breakers and I was wondering what you thought about the voting process called Alternative Vote?

The premise behind this vote is that the voters will be able to rank their choice from most wanted to least wanted (you can do all candidates or just up to a certain number).
After everyone has placed their rankings, the votes for all top choices are tallied.
If someone has won a majority (or 2/3, 3/4, etc.; whatever the limit is), then they win the position
If no one has one, the candidate with the least amount of votes is dropped from the candidate pool
All of the votes that originally went to this candidate are removed, and people who voted for a secondary candidate have their secondary candidate votes added in.
In the event that they only voted for their top choice, their vote is removed from the total number of votes.
The process is repeated until someone has won the limit set for the vote (majority, 2/3, etc.)


This method of voting has some extreme advantages:
* It discourages people from not voting for someone because they don't think they have a chance to win (in the example of the US, this would be like people who want to vote for a third party, but change their vote to their less desirable candidate because they think the third party can't win)
* It allows people to still generally get a candidate they are relatively okay with, as long as they are in the majority of people. So if they get their third choice, at least they didn't get their sixth, etc.